src/api/libcellml/variable.h

```diff
@@ -251,20 +264,22 @@ class LIBCELLML_EXPORT Variable: public NamedEntity
      * @deprecated This method is currently needed by our Printer and Validator classes, but this is
      * not a method that a libCellML user should ever need. It will therefore be removed at some point.
      */
-    bool hasDirectEquivalentVariable(const VariablePtr &equivalentVariable) const;
+    bool hasEquivalentVariable(const VariablePtr &equivalentVariable) const;
```

Comment:
is this method still deprecated?

Reply:
Probably not, may also look to add a parameter to this method and remove the need for `hasIndirectEquivalentVariable`.